lib/msal-core/src/error/ClientConfigurationError.ts

```diff
@@ -23,17 +23,22 @@
 
 import { Constants } from "../Constants";
 import { ClientAuthError } from "./ClientAuthError";
+import { AuthError } from "./AuthError";
 
 export const ClientConfigurationErrorMessage = {
     invalidCacheLocation: {
         code: "invalid_cache_location",
         desc: "The cache contains multiple tokens satisfying the requirements. " +
             "Call AcquireToken again providing more requirements like authority."
     },
-    noCallback: {
-        code: "no_callback",
-        desc: "Error in configuration: no callback(s) registered for login/acquireTokenRedirect flows. " +
-            "Please call handleRedirectCallbacks() with the appropriate callback signatures. " +
+    noSuccessCallback: {
```

Comment:
I am a really big fan about how the splitting of these error classes makes them readible all within one file